packages/react/src/patterns/blocks/ContentGroupHorizontal/ContentGroupHorizontal.js

```diff
@@ -20,20 +19,26 @@ const { prefix } = settings;
  */
 const ContentGroupHorizontal = ({ heading, items }) => {
   return (
-    <ContentBlock
-      heading={heading}
-      autoid={`${stablePrefix}--content-group-horizontal`}
+    <div
+      data-autoid={`${stablePrefix}--content-group-horizontal`}
       className={`${prefix}--content-group-horizontal`}>
```

Comment:
@greeshmareddyg just checking, is there a reason why a wrapper div was added around the `ContentBlock`?

cc: @kennylam

Reply:
@jeffchew No, earlier we have applied the classname if to the content block, as we are using within the contenthorizontalgroup I just have wrapped and applied styles. shall I apply to contentblock as before?

Reply:
@jeffchew This is actually the correct implementation. The Drupal team requested this change a while back because targeting the auto-id within `ContentBlock` itself was causing issues for them and suggested we lift it a level.

Reply:
Got it, thanks @greeshmareddyg @kennylam